users/static/user_site/day_view.css

```diff
@@ -0,0 +1,184 @@
+.dayview-container {
+    position: relative;
+    display: flex;
+    overflow: hidden;
+    align-items: stretch;
+    flex: 1 1 auto;
+
+    background-color: rgba(255, 255, 255, 0.3);
+    -webkit-backdrop-filter: blur(10px);
+    backdrop-filter: blur(10px);
+}
+.dayview-timestrings-container {
+    height: auto;
+    overflow-y: hidden;
+    flex: none;
+    display: flex;
+    align-items: flex-start;
+    min-width: 40px;
+}
+.dayview-timestring-container {
+    height: 40px;
+    position: relative;
+    padding-inline-end: 8px;
+    text-align: right;
+    padding-left: 5px;
+    border-left: white 1px solid;
+}
+
+.dayview-timestring-container:first-child {
+    border-top: white 1px solid;
+}
+
+.dayview-timestring-container:last-child {
+    padding-bottom: 41px;
+    border-bottom: white 1px solid;
+}
+
+.dayview-timestring-container:first-child .dayview-timestring {
+    display: none;
+}
+.dayview-timestring {
+    display: block;
+    /* color: #70757a; */
+    color: white;
+    font-size: 15px;
+    position: relative;
+    top: -10px;
+}
+.dayview-timestrings {
+    position: relative;
+    /* background-color: #fff; */
+    box-sizing: border-box;
+    margin-left: auto;
+}
+.dayview-grid-container {
+    flex: 1 1 0;
+    overflow-x: auto;
+    /* overflow-y: scroll; */
+    overflow-y: hidden;
```

Comment:
Please remove the comment lines if they are not needed.

Reply:
done. thanks 😄